.appveyor.yml

```diff
@@ -87,9 +89,13 @@ matrix:
     - PIP: master
 
 install:
-    - C:\Python36\python -m pip install tox
+    - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+    - pip install tox codecov
 
 build: false
 
 test_script:
-    - C:\Python36\Scripts\tox
+    - tox
+
+after_test:
+    - codecov
```

Comment:
```suggestion
    - codecov --flags windows --name %TOXENV%
```

Reply:
Good point. I think `--flag=linux` should be added also in the `.travis.yml`. What do you think?

Reply:
Yes, in general - but flags might trigger timeouts with codecov.io, so I would rather only use name here for now.
It does not post comments anyway, does it?  (where the flags would be visible)